tests/modules/nf-core/deseq2/differential/test.yml

```diff
@@ -4,21 +4,21 @@
     - deseq2
     - deseq2/differential
   files:
-    - path: output/deseq2/treatment-mCherry-hND6-sample_number.deseq2.results.tsv
+    - path: output/deseq2/treatment_mCherry_hND6_sample_number.deseq2.results.tsv
       md5sum: 86f0312a4691dfb4d7c15643059d8fb8
-    - path: output/deseq2/treatment-mCherry-hND6-sample_number.deseq2.sizefactors.tsv
+    - path: output/deseq2/treatment_mCherry_hND6_sample_number.deseq2.sizefactors.tsv
       md5sum: 6332f21889ecac387bb12eeb04f23849
-    - path: output/deseq2/treatment-mCherry-hND6-sample_number.normalised_counts.tsv
+    - path: output/deseq2/treatment_mCherry_hND6_sample_number.normalised_counts.tsv
       md5sum: 827cdf01f366d72d6e4b500e9862ba74
-    - path: output/deseq2/treatment-mCherry-hND6-sample_number.rlog.tsv
+    - path: output/deseq2/treatment_mCherry_hND6_sample_number.rlog.tsv
       md5sum: 1077cf03a8c6a155932eaaed7415b74c
-    - path: output/deseq2/treatment-mCherry-hND6.deseq2.results.tsv
+    - path: output/deseq2/treatment_mCherry_hND6_.deseq2.results.tsv
```

Comment:
This correct there is a trailing `_` after hND6?

Reply:
Yes- in this instance the id was generated dynamically by '_'-separating the elements of the contrast, and the blocking variable field was empty.